js-exercises/counting-anagrams/countingAnagrams.js

```diff
@@ -0,0 +1,26 @@
+const countingAnagrams = (str) => {
+    const tempAnagrams = {}
+    let flag = 0
+    str.forEach((loopingArrayWord)=>{
+      const wordAfterSorting = returnTheWordInSorted(loopingArrayWord)
+      if(tempAnagrams[wordAfterSorting])
+      {
+          if(wordAfterSorting.length > 1)
+          flag++
+          return tempAnagrams[wordAfterSorting].push(loopingArrayWord)
+      }
+      tempAnagrams[wordAfterSorting] = loopingArrayWord
+    })
+   return flag;
+};
+
+function returnTheWordInSorted(word) {
```

Comment:
`returnTheWordInSorted()` is not a good function name ... `sortWord()` could be better